### PR TITLE
Updated redirects to new docs

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: src/Octopurls/redirects.json [hunk 1]
```diff
@@ -1,131 +1,131 @@
...
+  "NuGetPackageHostingOptions": "http://octopusdeploy.com/documentation/packaging",
+  "WhatIsIncludedInAnonymousUsageStatistics": "http://octopusdeploy.com/documentation/security/outbound",
+  "Suggestions": "https://octopusdeploy.uservoice.com/",
+  "TrainingVideos": "http://octopusdeploy.wistia.com/projects/gguvhmqn6b",
+  "GettingStarted": "https://octopus.com/docs/getting-started",
+  "Upgrade2To3": "https://octopus.com/docs/administration/upgrading/upgrading-from-octopus-2.6",
+  "HighAvailability": "https://octopus.com/docs/administration/high-availability",
+  "SecurityAndEncryption": "https://octopus.com/docs/reference/security-and-encryption",
+  "NuGetVersioning": "https://docs.nuget.org/create/versioning",
+  "ChannelVersionRules": "https://octopus.com/docs/key-concepts/projects/channels#Channels-versionrules",
```
Comment (robpearson) [inline src/Octopurls/redirects.json]: Anchor should be #Channels-DefiningVersionRulesversionrules

File: src/Octopurls/redirects.json [hunk 2]
```diff
@@ -1,131 +1,131 @@
...
+  "ChannelVersionRuleTags": "https://octopus.com/docs/key-concepts/projects/channels#Channels-tags",
```
Comment (robpearson) [inline src/Octopurls/redirects.json]: Same thing. Needs updating to the new style.

File: src/Octopurls/redirects.json (hunk 1 above)
Comment (robpearson) [inline src/Octopurls/redirects.json]: I fixed this in the page.

File: src/Octopurls/redirects.json (hunk 2 above)
Comment (robpearson) [inline src/Octopurls/redirects.json]: I fixed this in the page.

File: src/Octopurls/redirects.json [hunk 3]
```diff
@@ -1,131 +1,131 @@
...
+  "StandaloneScripts": "https://octopus.com/docs/deploying-applications/custom-scripts/standalone-scripts",
+  "WhatIsEAP": "https://octopus.com/downloads/compare",
+  "UpgradeDenied": "https://octopus.com/downloads/compare",
+  "Artifacts": "https://octopus.com/docs/deploying-applications/artifacts",
+  "MultiTenantDeployments": "https://octopus.com/docs/guides/multi-tenant-deployments",
+  "MultiTenantVariables": "https://octopus.com/docs/guides/multi-tenant-deployments/multi-tenant-deployment-guide/working-with-tenant-specific-variables",
+  "MultiTenantTags": "https://octopus.com/docs/guides/multi-tenant-deployments/multi-tenant-deployment-guide/working-with-groups-of-tenants-using-tags",
+  "MultiTenantProjectTenantRequired": "https://octopus.com/docs/guides/multi-tenant-deployments/multi-tenant-deployments-faq#Multi-tenantdeploymentsFAQ-CanIrequireatenantforalldeploymentsofaproject?",
+  "MultiTenantUntenantedDeploymentTarget": "https://octopus.com/docs/guides/multi-tenant-deployments/multi-tenant-deployments-faq#Multi-tenantdeploymentsFAQ-CanIconfiguredeploymenttargetstoworkwithuntenanteddeployments?",
```
Comment (robpearson) [inline src/Octopurls/redirects.json]: This one doesn't work.

File: src/Octopurls/redirects.json [hunk 4]
```diff
@@ -1,131 +1,131 @@
...
+  "MultiTenantHostingModel": "https://octopus.com/docs/guides/multi-tenant-deployments/multi-tenant-deployment-guide/designing-a-multi-tenant-hosting-model",
+  "MultiTenantDeploySimpleMultiTenantProject": "https://octopus.com/docs/guides/multi-tenant-deployments/multi-tenant-deployment-guide/deploying-a-simple-multi-tenant-project",
+  "MigrateAzureTargets": "http://docs.octopus.com/display/OD/Migrate+Azure+Targets+into+Azure+Steps",
+  "AutoDeployTroubleshooting": "https://octopus.com/docs/guides/elastic-and-transient-environments/keeping-deployment-targets-up-to-date#Keepingdeploymenttargetsuptodate-Troubleshooting",
```
Comment (robpearson) [inline src/Octopurls/redirects.json]: This one doesn't work.

File: src/Octopurls/redirects.json (hunk 3 above)
Comment (johnsimons) [inline src/Octopurls/redirects.json]: fixed

File: src/Octopurls/redirects.json (hunk 4 above)
Comment (johnsimons) [inline src/Octopurls/redirects.json]: fixed

**Pull request merged**